test/test.mk

```diff
@@ -19,16 +18,14 @@ endif
 
 GO_TEST_FLAGS := 			 --format=$(GO_TEST_FORMAT) --junitfile $(REPORTS)/junit_unit_test.xml $(GOTEST_PUBLISH_FLAGS)
 GO_TEST_INTEGRATION_FLAGS := --format=$(GO_TEST_FORMAT) --junitfile $(REPORTS)/junit_integration_test.xml $(GOTEST_PUBLISH_FLAGS)
-GO_TEST_E2E_FLAGS := 		 --format=$(GO_TEST_FORMAT) --junitfile $(REPORTS)/junit_e2e_test.xml $(GOTEST_PUBLISH_FLAGS)
 
 _integration_test: $(REPORTS)
 	gotestsum $(GO_TEST_E2E_FLAGS) -- $(GO_INTEGRATIONTEST_FLAGS) -timeout $(TIMEOUT) || ($(MAKE) _collect_junit && /bin/false)
 	$(MAKE) _collect_junit
 
 _e2e_test: $(REPORTS)
 	sudo chown $(shell whoami):$(shell whoami) -R bin/output
-	gotestsum $(GO_TEST_E2E_FLAGS) -- $(GO_E2E_FLAGS) -timeout $(TIMEOUT) || ($(MAKE) _collect_junit && /bin/false)
-	$(MAKE) _collect_junit
+	ginkgo run --timeout 30m --race -vv --junit-report $(REPORTS)/junit_e2e_test.xml --github-output $(GO_E2E_DIRS)
```

Comment:
I switched to ginkgo runner because it provides better output of what's going on, and better junit output too, we may consider also switching the integration test output for the same reason.